src/pages/casting.tsx

```diff
@@ -1,15 +1,41 @@
 import type { NextPage } from 'next';
 import Head from 'next/head';
+import { useEffect, useState } from 'react';
 
-import { H1 } from '@/components/Text';
+import Search from '@/components/Search/index';
+import Card from '@/components/Card/index';
+import Button from '@/components/Button';
+
+interface Props {
+  id: number;
+  groupName: string;
+  name: string;
+  squreImageUrl: string;
+  longImageUrl: string;
+  debutDate: string;
+}
 
 const Casting: NextPage = () => {
+  const [data, setData] = useState<Props[]>([]);
```

Comment:
```suggestion
  const [data, setData] = useState<DataProps[]>([]);
```
props 명을 더 명확하게 지어주면 좋을 것 같아요